index.html

```diff
@@ -11,23 +11,15 @@
   <header>
 		<h1 class="main-header">Awesome books</h1>
 	</header>
-	<ul class="books-list">
-		<ul class="book-list">
-			<li class="book-title">First Book</li>
-			<li class="book-author">sit amet</li>
-      <button class="remove-btn" type="reset">Remove</button>
-			<hr>
-		</ul>
-		<ul class="books-list">
-			<li class="book-title">Second Book</li>
-			<li class="book-author">sit amet</li>
-			<button class="remove-btn" type="reset">Remove</button>
-			<hr>
-		</ul>
-	</ul>
-  <form action="" class="add-book-form">
-		<label for="title"></label>
-    <input type="text" placeholder="Title" name="title" class="">
+	  <div id="display-book-list"></div>
+  <form action="" class="add-book-form" id="add-form">
+		<label for="title">
+    	<input type="text" placeholder="Title" name="title" id="title" class="title-input">
+		</label>
+		<label for="author">
+  		<input type="text" placeholder="Author" name="author" id="author" class="author-input">
+    </label>
```

Comment:
- [ ] [optional] Kindly move this label to align with the opening tab, it makes your code look neat.